src/Controller/Api/ProjectController.php

```diff
@@ -77,4 +79,20 @@ public function project(string $id, ProjectRepositoryInterface $repository, Proj
 
         return new SuccessResponse($converter->full($project));
     }
+
+    /**
+     * @param string $id
+     * @param ProjectRepositoryInterface $repository
+     * @return SuccessResponse
+     * @throws \App\Project\Exception\InvalidProjectIdException
+     * @Route("/remove/{id}")
+     */
+    public function remove(string $id, ProjectRepositoryInterface $repository)
+    {
+        //TODO: some checks maybe?
```

Comment:
Yes, add method to Model\Project canRemove (with logic like canManageUsers for now). And check against it. Throw Access denied if cannot.